spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe StripeWrapper do
+  describe StripeWrapper::Charge do
+    describe ".create" do
+      it "makes a successful charge", :vcr do
+        token = Stripe::Token.create(
+          :card => {
+            :number => "4242424242424242",
+            :exp_month => 11,
+            :exp_year => 2017,
```

Comment:
I would recommend to use dynamic year here, something like `Time.now.year + 1`, if you happen to hit 2017 and 11 it might be a problem and your specs will fall.